ietf/iesg_statement/templates/iesg_statement/iesg_statement_page.html

```diff
@@ -1,231 +1,79 @@
-                            <p>Show all</p>
-                        </div>
-                    </a>
-                </li>                
-            </ul>
-        </div>
-    </div>
-</div>
-
-
-{# highlights mobile #}
-<div class="highlights push-up cf mobile-show-tablet">
+<div class="bg-white">
     <div class="container">
-        <div class="eight none">
-            <a class="blog-pagination" href="{{ self.previous.url }}">
-                <div class="highlight smaller">
-                    <div class="text">
-                        <p class="sub">
-                            <h3 class="small">Previous</h3>{{ self.previous.title }}
-                        </p>
-                    </div>
+        <div class="row flex-row-reverse align-items-stretch no-gutters">
+            <aside class="col-12 col-lg-4 d-flex flex-column bg-light">
+                <div class="d-none d-lg-block">
+                    {% include "includes/blog_sidebar.html" with siblings=siblings current=self %}
                 </div>
-            </a>
-            <a class="blog-pagination" href="{{ self.next.url }}">
-                <div class="highlight smaller">
-                    <div class="text">
-                        <p class="sub">
-                           <h3 class="small">Next</h3>{{ self.next.title }}
-                        </p>
-                    </div>
+                <button class="btn btn-outline-primary d-lg-none w-100" type="button" data-toggle="collapse"
+                    data-target="#blog_sidebar" aria-expanded="false" aria-controls="collapseExample">
+                    Statement listing
+                    <span class="icon ion-ios-arrow-down"></span>
+                </button>
+                <div id="blog_sidebar" class="collapse d-lg-none">
+                    {% include "includes/blog_sidebar.html" with siblings=siblings current=self %}
                 </div>
-            </a>
-        </div>
-    </div>
-</div>
-{# highlights desktop #}
-<div class="highlights">
-    <div class="row container">
-        <div class="eight">
-            <div class="highlights tablet-desktop">
-                <a class="blog-pagination" href="{{ self.previous.url }}">
-                    <div class="highlight smaller">
-                        <div class="text">
+            </aside>
+            <main role="main" class="col-12 col-lg-8">
+                <div class="px-3 px-xl-5 mb-5">
+                    {% include 'includes/breadcrumbs.html' %}
+                    
+                    {% include "includes/blog_filters.html" with topics=topics parent_url=parent_url %}
+                    {% if filter_text %}
+                        <p class="h6">{{ filter_text }}</p>
+                    {% endif %}
+                    <h1>{{ self.title }}</h1>
+                    <p>{{ self.date|date:"DATE_FORMAT" }}</p>
+                    
+                    <p class="intro">{{ self.introduction }}</p>
```

Comment:
```suggestion
                    <p class="lead">{{ self.introduction }}</p>
```
Probably

Reply:
how did i manage to not copy and paste that bit 🤔